src/api/window.js

```diff
@@ -18,27 +18,9 @@ export function initLayout(
   `);
 }
 
-export function getAttributesInstance(
-  attrType,
-  tmpId,
-  docType,
-  docId,
-  tabId,
-  rowId,
-  fieldName,
-  entity
-) {
-  const type = entity === 'process' ? 'processId' : 'windowId';
-
+export function getAttributesInstance(attrType, tmpId) {
```

Comment:
hmmm, have to check this.... 
I mean, it would work for /address endpoint but not for /attributes endpoint.

On server side we are handling them differently but on client side it seems they are handle it the same way...
Will check and update here.

Reply:
update: introduced the `source` for /address endpoint too (commit https://github.com/metasfresh/metasfresh-webui-api/commit/266ededcb9ed38fec05d43a39092d8d30979495d).

So, pls revert this and allow sending the source too.